src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -248,10 +238,14 @@ const ArticlePage = ({ pageData, mostReadEndpointOverride }) => {
         {!isApp && <SecondaryColumn pageData={pageData} />}
       </div>
       {!isApp && (
-        <MostReadContainer
-          mostReadEndpointOverride={mostReadEndpointOverride}
-          wrapper={MostReadWrapper}
-        />
+        <MostReadSection css={styles.mostReadSection}>
```

Comment:
We don't need to add this section - it's already added in the MostRead component.

You should just be able to apply the css directly to the MostRead component